sunkit_magex/pfss/input.py

```diff
@@ -23,8 +23,9 @@ class Input:
         on.
     rss : float
         Radius of the source surface, as a fraction of the solar radius.
-    br_outer : sunpy.map.GenericMap, str
+    br_outer : sunpy.map.GenericMap, str, optional
```

Comment:
i think this is tripping the docs build, the syntax might need to be 

`sunpy.map.GenericMap or str, optional`

or 

`Optional[sunpy.map.GenericMap or str]` 

are you able to test the docs build locally?

Reply:
Oh yeah, my suggestion was wrong.

Reply:
Yes, I can test the docs build locally. I have accidentally been doing so on a separate, fresh download of sunkit-magex, so I have been missing these errors. Hence why I pushed 98b2105. I did not notice this until now. My apologies.